lib/ui/views/signup/signup_view.dart

```diff
@@ -0,0 +1,181 @@
+                textInputAction: TextInputAction.next,
+                decoration: const InputDecoration(
+                    labelText: "Last Name", hintText: "Last Name"),
+              ),
+              SizedBox(
+                height: 16.h,
+              ),
+              TextFormField(
+                controller: emailController,
+                focusNode: emailFocusNode,
+                autofillHints: const [AutofillHints.email],
+                keyboardType: TextInputType.emailAddress,
+                textInputAction: TextInputAction.next,
+                validator: Validation.validateEmail,
+                decoration: const InputDecoration(
+                    labelText: "Email Address", 
+                    hintText: "Email Address"),
+              ),
+              SizedBox(
+                height: 16.h,
+              ),
+              TextFormField(
+                controller: passwordController,
+                focusNode: passwordFocusNode,
+                obscureText: viewModel.hidePassword,
+                  textInputAction: TextInputAction.done,
+                validator: Validation.validateField,
+                decoration: InputDecoration(
+                  labelText: "Password",
+                  hintText: "Password",
+                  suffixIcon: IconButton(
+                    onPressed: viewModel.togglePassword,
+                    icon: Icon(
+                      viewModel.hidePassword
+                          ? Icons.visibility
+                          : Icons.visibility_off,
+                    ),
+                  ),
+                ),
+              ),
+              SizedBox(
+                height: 129.h,
+              ),
+              PrimaryButton(
+                buttonText: "Sign up",
+                onTap: () {
+                  if (_signupKey.currentState?.validate() ?? false) {
+                    viewModel.signup();
+                  }
+                },
+              ),
+              SizedBox(
+                height: 16.h,
+              ),
+              Text.rich(TextSpan(
+                  text: "Already have an account?",
+                  style: typography?.titleRegular16
+                      ?.copyWith(color: palette?.gray8, fontSize: 14.sp),
+                  children: [
+                    const TextSpan(text: " "),
+                    TextSpan(
+                        text: "Login",
+                        style: typography?.titleBold16?.copyWith(
+                          color: palette?.primary6,
+                          fontSize: 14.sp,
+                        ),
+                        recognizer: TapGestureRecognizer()
+                          ..onTap = viewModel.actionRouteToLoginView)
+                  ]))
+            ],
+          ),
+        ),
+      ),
+    );
+  }
+
+   @override
+  void onViewModelReady(SignupViewModel viewModel) {
+    syncFormWithViewModel(viewModel);
```

Comment:
where is ur `super` call here?

Reply:
it is with the dispose now

Reply:
nope, its a separate logic on its own, so its meant to have it's super called as well.

Reply:
this is still missing its `super` call.

Reply:
nothing like that in the login screen